src/storage.cc

```diff
@@ -159,9 +244,7 @@ void Storage::Tock() {
   result = std::max_element(in_commod_prefs.begin(), in_commod_prefs.end());
   int maxindx = std::distance(in_commod_prefs.begin(), result);
   double demand = 0;
-  if (manager()->context()->time() % (active_buying + dormant_buying) < active_buying) {
```

Comment:
What does removing this do? It looks like it means the demand is set at current capacity even if there is space for it.

Reply:
This is basically overriding my first attempt at an active and dormant policy. In #546, I added the ability to cycle back and forth between two states: active (regular behavior) and dormant (no demand for new material, regardless of inventory), staying in each state for a fixed number of time steps. However, when I went to expand this behavior to include randomness (like active length sampled from a uniform distributions between 1 and 5 time steps), @gonuke recommended I completely re-do the way that these distributions are created and managed.

In this old version (that I am now removing), Storage was the one calculating whether or not it was active. The lines above show that now, all you do is create the underlying distributions, init the buy policy, and the buy policy takes care of the rest. Archetypes don't have to do that sampling themselves.